sources/api/migration/migrations/v1.11.0/kubelet-tls-config-files/src/main.rs

```diff
@@ -0,0 +1,42 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::{ListReplacement, ReplaceListsMigration};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// New configuration files for kubelet TLS cert configuration.
+fn run() -> Result<()> {
+    migrate(ReplaceListsMigration(vec![ListReplacement {
+        setting: "services.kubernetes.configuration-files",
+        old_vals: &[
+            "kubelet-env",
+            "kubelet-config",
+            "kubelet-kubeconfig",
+            "kubelet-bootstrap-kubeconfig",
+            "kubelet-exec-start-conf",
+            "kubernetes-ca-crt",
+            "proxy-env",
+        ],
+        new_vals: &[
+            "kubelet-env",
+            "kubelet-config",
+            "kubelet-kubeconfig",
+            "kubelet-bootstrap-kubeconfig",
+            "kubelet-exec-start-conf",
+            "kubernetes-ca-crt",
+            "proxy-env",
+            "kubernetes-tls-crt",
+            "kubernetes-tls-private-crt",
+        ],
```

Comment:
Just a reminder that we'll somehow need to sync up this list with #2377, which adds "credential-provider-config-yaml" as well, or else ensure that the migrations run in a specific order.

(Maybe rename this to a shared migration called `kubelet-new-config-files`.)